ChangeLog

```diff
@@ -1,3 +1,11 @@
+Changes in squid-6.9 (7 Apr 2024)
+	- Bug 5069: Keep listening after getsockname() error
+	- Reduce stale errno usage
+	- Plug memory leak in handling cache manager requests
+	- Fix NIS helper header-guard
```

Comment:
Several things missed here:
 1) For binaries other than `squid` we name it specifically to be clear of the change relevance, and
 2) this has a buzilla report that needs referencing, and
 3) this is a known regression since earlier 6.7 release.

```suggestion
	- Regression Bug 5349: basic_nis_auth build error: unterminated #ifndef
```
... being a Regression this should be sorted first, above the "Bug ..." lines.